### ISSUE TITLE
[Charset Storage - Locks] Visual bug.

### ISSUE THREAD

**Issue by lublak** [asiekierka/CharsetMC#424]
On a doublesized chest the lock is not centered.

![grafik](https://user-images.githubusercontent.com/44057030/63218536-1631ea00-c15d-11e9-8fbd-d2668610ed81.png)

**Comment by asiekierka** [comment]
Yes, because the lock is placed on the specific block, technically speaking. Not likely to be fixed any time soon, unfortunately.
